mpt_trie/examples/simple.rs

```diff
@@ -43,8 +44,10 @@ fn main() {
     );
 
     // Values can be deleted:
-    let del_val = trie.delete(0x1234_u32);
-    assert_eq!(del_val, Some(b"some data".to_vec()));
+    let del_val = trie
+        .delete(0x1234_u32)?
+        .expect("Failed to delete from trie");
+    assert_eq!(del_val, b"some data".to_vec());
```

Comment:
nit: I preferred the old `assert_eq!(del_val, Some(b"some data".to_vec()));` since the result of `delete` can be `None` without having "Failed to delete from trie", for example if the key is not in the trie.